src/js/msp/MSPHelper.js

```
@@ -1728,6 +1789,10 @@ MspHelper.prototype.process_data = function (dataHandler) {
                 case MSPCodes.MSP_MULTIPLE_MSP:
                     let hasReturnedSomeCommand = false; // To avoid infinite loops
```

Biome (2.4.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)